packages/svelte-undp-design/src/lib/Table/Table.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  export let variant: "default" | "striped" | "border" = "default";
+  export let size: "small" | "large" = "small";
+
+  export let data = [
+    {
+      id: 1,
+      name: "John Doe",
+      email: "john.doe@gmail.com",
+      optional_html: '<button id="delete-1" class="delete">Click me</button>'
```

Comment:
other than adding html string, we may also add HTML DOM element. maybe either HTML string of DOM element can be specified as optional